src/plugins/multitrack-media/multitrack-media.less

```diff
@@ -1,3 +1,13 @@
 .MultitrackMediaDisplay {}
 
+.MultitrackMediaDisplay-content {
+  margin: 0 auto;
+}
+
+.MultitrackMediaDisplay-errorMessage {
+  font-weight: bold;
```

Comment:
From my side I think we can drop this, makes the UI a bit too aggressive 🤔

Reply:
True, especially as this is kind of the initial default view. Dropped.